README.md

```diff
@@ -2,14 +2,12 @@
 This analysis retrieves the device list of your account and print to the console.
 
 ## How to use on TagoIO
-Environment Variables<br>
-In order to use this analysis, you must setup the Environment Variable table.<br>
 
-account_token: Your account token
+In order to use this analysis, you must to add a new policy in your account.<br>
 
-Steps to generate an account_token:<br>
-1 - Enter the following link: https://admin.tago.io/account/<br>
-2 - Select your Profile.<br>
-3 - Enter Tokens tab.<br>
-4 - Generate a new Token with Expires Never.<br>
-5 - Press the Copy Button and place at the Environment Variables tab of this analysis.<br>
+
+Steps to add a new policy:
+   1 - Click in the button "Add Policy" in this url: https://admin.tago.io/am;
+   2 - In the Target select Analysis with the field Id and choose your analysis in the list;
+   3 - Click in "Click to add a new permission" and select "Device" with rules "Access" with the field "Any";
```

Comment:
3 - Click the  "Click to add a new permission" element and select "Device" with the rule "Access" with the field as "Any";